theme_anelusia/views/images_library.xml

```diff
@@ -20,7 +20,7 @@
 <record id="s_text_cover_default_image" model="theme.ir.attachment">
     <field name="key">website.s_text_cover_default_image</field>
     <field name="name">website.s_text_cover_default_image</field>
-    <field name="url">website.s_banner_default_image</field>
+    <field name="url">/theme_anelusia/static/src/img/snippets/s_banner.jpg</field>
```

Comment:
As I said in DMs, I wonder if this is a wrong URL. We'll have to look up whether this is ok to have or not, because I assume your test flagged it as wrong? In any case, it's not bad if we enforce some uniformity to those records anyway.